include/deal.II/grid/grid_tools.h

```diff
@@ -146,7 +146,34 @@ namespace GridTools
    * Compute the volume (i.e. the dim-dimensional measure) of the
    * triangulation. We compute the measure using the integral $\sum_K \int_K 1
    * \; dx$ where $K$ are the cells of the given triangulation. The integral
-   * is approximated via quadrature for which we need the mapping argument.
+   * is approximated via quadrature. This version of the function uses a
+   * linear mapping to compute the JxW values on each cell.
+   *
+   * If the triangulation is a dim-dimensional one embedded in a higher
+   * dimensional space of dimension spacedim, then the value returned is the
+   * dim-dimensional measure. For example, for a two-dimensional triangulation
+   * in three-dimensional space, the value returned is the area of the surface
+   * so described. (This obviously makes sense since the spacedim-dimensional
+   * measure of a dim-dimensional triangulation would always be zero if dim @<
+   * spacedim).
+   *
+   * This function also works for objects of type
+   * parallel::distributed::Triangulation, in which case the function is a
+   * collective operation.
+   *
+   * @param tria The triangulation.
+   * @return The dim-dimensional measure of the domain described by the
+   * triangulation, as discussed above.
+   */
```

Comment:
Can't you use `copydoc` here?

Reply:
Since one function takes a mapping argument and the other does not these strings are slightly different so I don't think `@copydoc` will work.

Reply:
Fair enough.